src/Helpers/Utils.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Godot\AssetLibrary\Helpers;
 
+use PDO;
```

Comment:
Is this needed? routes/assets.php seems to not require it.

Reply:
I had to remove it, since it didn't run on tuxfamily with it (they have some ancient php versions there).